test/quantization/test_quantize_script.py

```diff
@@ -1672,17 +1669,15 @@ def forward(self, x):
         m1 = convert_script(m, debug=True)
         conv_op_quant = 4
         # NB: This Needs to be updated when we add more ops to test
-        general_value_op_quant = 2
-        FileCheck().check_count("aten::quantize_per_tensor", conv_op_quant + general_value_op_quant + 1, exactly=True) \
+        general_value_op_quant = 12
+        FileCheck().check_count("aten::quantize_per_tensor(", conv_op_quant + general_value_op_quant + 1, exactly=True) \
```

Comment:
How does the IR look here? Is it conv->q->dq->avg_pool->q->dq->avg_pool-.>?

Reply:
yes